tests/integration/kubernetes/runtimeclass_workloads/confidential/unencrypted/ssh/unencrypted

```diff
@@ -0,0 +1,7 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
Sorry, I know this one was merged all ready. I was thinking the ssh folder could live two folder levels up so that the key (or anything else related) could be used by any test, not just the confidential unencrypted one. For that matter the Dockerfile above doesn't have anything confidential or unencrypted related either - although maybe it might later?

Reply:
Another thought is that the key could just get generated on the fly in the script that generates the image, and then we wouldn't have a key under source control. I don't think it's a big deal, but someone may be curious as to why we have keys under source control in a security project. The understanding here of course is that the key is for testing purposes only.